pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -21,6 +21,8 @@ const (
 
 	RouterUsername   = "system:" + RouterUnqualifiedUsername
 	RegistryUsername = "system:" + RegistryUnqualifiedUsername
+
+	BuildControllerUsername = "system:serviceaccount:openshift-infra:build-controller"
```

Comment:
see https://github.com/liggitt/origin/commit/7b0539fe9bbb7658fb19bf23e86bd9154022947e as an example for getting the name of the build controller service account into the bootstrap SCC